docs_source_files/content/getting_started/quick.de.md

```diff
@@ -26,13 +26,7 @@ vertraut zu machen.
 
 ## Grid
 
-Selenium Grid allows you to run test cases in different 
-machines across different platforms. The control of 
-triggering the test cases is on the local end, and 
-when the test cases are triggered, they are automatically 
-executed by the remote end.
-
-After the development of the WebDriver tests, you may face 
-the need of running your tests on multiple browser and 
-operating system combinations.
-This is where _[Grid]({{< ref "/grid/_index.md" >}})_ comes into the picture.
\ No newline at end of file
+Mit Selenium Grid können Tests auf verschiedenen Maschinen auf unterschiedlichen Plattformen ausgeführt werden. 
```

Comment:
Hi @f-g-s,

Thanks for the PR. Can you please make sure to have a line with a max of 70-80 characters? You can split it into multiple lines if possible.

Thanks,
Harsha